db/schema.rb

```diff
@@ -235,6 +236,10 @@
     t.string "amount", null: false
     t.string "fees", null: false
     t.string "url"
+    t.string "uphold_status_was"
+    t.boolean "reauthorization_was_needed", null: false
```

Comment:
Would you update the schema to make sure `null: false` isn't included?

Reply:
good catch!